app/views/validation_requests/index.html.erb

```diff
@@ -22,7 +24,7 @@
 </div>
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
-    <ol class="moj-task-list">
+    <ol class="govuk-list govuk-list--number">
```

Comment:
Should this change to `<ul>`? If the `govuk-list--number` is providing the numbering now, that is.

Reply:
Oh, never mind, I've misunderstood that.